app/models/video.py

```diff
@@ -1,4 +1,20 @@
 from app import db
 
+
 class Video(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    total_inventory = db.Column(db.Integer)
+    # Many to many: only counts each customer once. Secondary joins the third table
+    # backref - joins two tables
+    renters = db.relationship('Customer', secondary='rental')
+    # One to many. Counts for all videos currently checked out
+    rentals = db.relationship('Rental')
```

Comment:
This will add a `video` attribute to the Rental model

```suggestion
    rentals = db.relationship('Rental', back_ref = "video")
```